src/cdk-experimental/radio-group/radio-group.ts

```diff
@@ -162,35 +188,15 @@ export class CdkRadioGroup<V> {
     });
 
     afterRenderEffect(() => {
-      if (!this._hasFocused() && !this.toolbar) {
+      if (!this._hasFocused() && this.pattern instanceof RadioGroupPattern) {
```

Comment:
`!this._hasToolbar()` would be more clear than `this.pattern instanceof RadioGroupPattern`